glean-core/ios/Glean/Glean.swift

```diff
@@ -121,6 +122,10 @@ public class Glean {
                 _ = self.setDebugViewTag(debugViewTag)
             }
 
+            if let sourceTags = self.sourceTags {
```

Comment:
On Android we set logPings befors sourceTags in init.
While I don't think this has semantic meaning, it would be good to keep them in the same order, so reasoning about both code bases is easier.